ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -338,3 +338,11 @@ def close(self):
         if self.temp_credentials:
             os.unlink(self.temp_credentials)
         os.environ.pop("GOOGLE_CLOUD_PROJECT", "")
+
+    def test_connection(self) -> str:
+        """
+        Used a timed-bound function to test that the engine
+        can properly reach the source
+        """
+        test_connection_fn = get_test_connection_fn(self.service_connection)
+        test_connection_fn(self.engine, self.service_connection)
```

Comment:
should this return something?

Reply:
It should return None.
I will change this.